module/VuFind/src/VuFindTest/Feature/ConfigPluginManagerTrait.php

```diff
@@ -47,27 +48,67 @@ trait ConfigPluginManagerTrait
      * Get a mock configuration plugin manager with the given configuration "files"
      * available.
      *
-     * @param array $configs An associative array of configurations where key is the
-     * file (e.g. 'config') and value an array of configuration sections and
-     * directives
+     * @param array            $configs   An associative array of configurations
+     * where key is the file (e.g. 'config') and value an array of configuration
+     * sections and directives
+     * @param array            $default   Default configuration to return when no
+     * entry is found in $configs
+     * @param ?InvocationOrder $getExpect The expected invocation order for the get()
+     * method (null for any)
+     * @param ?InvocationOrder $hasExpect The expected invocation order for the has()
+     * method (null for any)
      *
-     * @return MockObject
+     * @return MockObject&PluginManager
      */
-    protected function getMockConfigPluginManager(array $configs): MockObject
-    {
+    protected function getMockConfigPluginManager(
+        array $configs,
+        array $default = [],
+        InvocationOrder $getExpect = null,
+        InvocationOrder $hasExpect = null
+    ): PluginManager {
```

Comment:
I thought it was better to typehint with the more specific type, but I included MockObject in a union type above.